package.json

```diff
@@ -29,6 +29,7 @@
   "homepage": "https://gitlab.com/biomage/api#readme",
   "dependencies": {
     "@kubernetes/client-node": "^0.12.3",
+    "@vscode/sqlite3": "^5.0.8",
```

Comment:
why do you need this?

Reply:
i had some errors when i was trying to make a new migration, so when i installed it, it worked. But now the build fails and i think its because of this so I'll remove it.